ui/app/components/section.ts

```diff
@@ -4,10 +4,12 @@ import { action } from '@ember/object';
 
 type Args = {
   expanded?: boolean;
+  isExpandable?: boolean;
 };
 
 export default class extends Component<Args> {
   @tracked expanded = this.args.expanded ?? true;
+  @tracked isExpandable = this.args.isExpandable ?? true;
```

Comment:
I’d be inclined to implement this with a getter, that way it’s a little clearer that this is a read-only property (not something we expect to change internally).